docs/applications/consdb/index.rst

```diff
@@ -0,0 +1,22 @@
+.. px-app:: consdb
+
+###########################################
+Consdb — populate the consolidated database
```

Comment:
```suggestion
Consdb — Populate the consolidated database
```

This capitalization needed for the documentation style

Reply:
that is really a bit OTT and the error in no way tells me that is the problem